packages/app-accounts/src/modals/Create.tsx

```diff
@@ -173,13 +175,14 @@ function Create ({ className, onClose, onStatusChange, seed: propsSeed, t, type:
   };
   const _onChangeName = (name: string): void => setName({ isNameValid: !!name.trim(), name });
   const _toggleConfirmation = (): void => setIsConfirmationOpen(!isConfirmationOpen);
+  const context = useContext<MyAccountContextProps>(MyAccountContext);
 
   const _onCommit = (): void => {
     if (!isValid) {
       return;
     }
 
-    const status = createAccount(`${seed}${derivePath}`, pairType, name, password, t('created account'));
+    const status = createAccount(context, `${seed}${derivePath}`, pairType, name, password, t('created account'));
```

Comment:
You can avoid passing account context to `createAccount` function by getting value of `status.account` after its execution.
```suggestion
    const status = createAccount(`${seed}${derivePath}`, pairType, name, password, t('created account'));
    if (status.account) {
      context.set(status.account);
    }
```